content/blog/intro-to-web-accessability/index.md

```diff
@@ -0,0 +1,301 @@
+
+You also need to make sure to [test your application](#testing) as you would any other part of your app. We'll touch on this more in a future section.
+
+If anyone is ever advertising to you that your project can be made accessible without any changes to your codebase: they're either lying to you or don't understand accessibility properly.
+
+## Assistance is Amicable {#eslint}
+
+While full automation will never be possible for improving a project's accessibility, not everyone proposing assistance in the process is trying to sell snake oil. 
+
+For example, [Deque's open-source Axe project](https://github.com/dequelabs/axe-core) can help identify issues such as common HTML semantic errors, contrast problems, and more. There are even libraries that help integrate Axe into your project's linters, such as one for React called [`eslint-plugin-jsx-a11y`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y).
+
+However, keep in mind that these tools are not infallible and are meant to supplement accessibility experts working with your engineering team, not replace them.
+
+# Test, Test, Test Again {#testing}
+
+Testing is a critical component of any application release. Whether using automated testing solutions or QA teams, they help ensure that your users are getting the best experience possible without regressions in an application's behavior.
+
+As a developer working on the front end, you should be regularly turning on a screen-reader to analyze your site and navigating your application [with only your keyboard](#keyboard). This will help enforce the feedback loop between building the functionality and getting it shipped to users. Make it part of your PR review process - have other members of your team test with a screen-reader as they would visually analyze new features.
+
+You're also able to include automated tests that will help with A11Y regressions. Either using [integration tests](https://kentcdodds.com/blog/write-tests) or e2e tests, you can use real-world behavior such as "search for an element with this label" or "tab over to this button" to ensure your application is functioning as intended.
+
+As mentioned in [a previous section](#no-automation), the process to make your app accessible cannot be fully automated. This extends to testing as well. While real-world automated tests are fine and well, you need someone to experience the application on a broader scale to make sure the experience is as fluid as it can be. While a specific component might be accessible by default, perhaps in specific usages, it falls flat.
```

Comment:
```suggestion
As mentioned in [a previous section](#no-automation), the process to make your app accessible cannot be fully automated. This extends to testing as well. While real-world automated tests are fine and well, you need someone to experience the application on a broader scale to make sure the experience is as fluid as it can be. While a specific component might be accessible by default, perhaps in specific usages, it falls flat. [Displaying an accessibility statement](https://www.w3.org/WAI/planning/statements/) while transforming your users' reported problems into bug tickets and performing user testing with disabled users are great ways to close the loop with the real people affected.
```

Reply:
I hadn't heard of A11Y statements before! I love this idea for UU in particular - where do you think we should display this statement? In our about page? In the README with a link to a dedicated `.md` file?